src/createVueApp.ts

```diff
@@ -21,5 +27,7 @@ export function createVueApp( rootComponent: Component, messages: Messages, root
 		};
 	}
 
+	app.component( 'FeatureToggle', createFeatureToggle( activeFeatures ) );
```

Comment:
There actually was an interface for it https://github.com/wmde/fundraising-app-frontend/blob/dd5d1bce1da914b1b2aa90e3bab9fdaaf1f64451/src/createFeatureToggle.ts#L3

But I purposely removed it because it was only ever implemented like this: https://github.com/wmde/fundraising-app-frontend/blob/dd5d1bce1da914b1b2aa90e3bab9fdaaf1f64451/src/pages/donation_form.ts#L84 IMO that was an unneeded abstraction, but I'm happy to revisit it once we know if we're going to be adding session storage.

Reply:
Okay, take a look at that new implementation

Reply:
I like it!